ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -112,8 +117,17 @@ export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQu
         super.ngOnDestroy();
         this.ngUnsubscribe.next();
         this.ngUnsubscribe.complete();
+        window.removeEventListener('scroll', this.onScroll, true);
     }
 
+    onScroll = (event): void => {
```

Comment:
This arrow function definition is technically correct, but looks out of place between instance methods.
Consider rewriting as `onScroll(event: WhateverTheEventType): void {`

Reply:
I used an arrow function to avoid ugly `this` binding. Will see what else I can do